dependencies.yaml

```diff
@@ -101,6 +101,18 @@ dependencies:
       - output_types: [requirements, pyproject]
         packages:
           - wheel
+      - output_types: [conda]
+        packages:
+          - boost-cpp
```

Comment:
```suggestion
          - libboost-headers
```

Reply:
Done in commit: https://github.com/rapidsai/cucim/pull/753/commits/fce7ab4b4b7e6f4b0d4999e2415d6f6dc89d7781